src/messages/MClientCaps.h

```diff
@@ -117,7 +120,8 @@ class MClientCaps : public Message {
       time_warp_seq(0),
       osd_epoch_barrier(0),
       oldest_flush_tid(0),
-      caller_uid(0), caller_gid(0) {
+      caller_uid(0), caller_gid(0),
+      sync(false) {
```

Comment:
Need to set HEAD_VERSION = 10 above, and make the decoding of the new field conditional on that